app/src/components/market/sections/market_sell/market_sell.tsx

```diff
@@ -5,7 +5,13 @@ import { RouteComponentProps, withRouter } from 'react-router-dom'
 import ReactTooltip from 'react-tooltip'
 import styled from 'styled-components'
 
-import { useAsyncDerivedValue, useConnectedCPKContext, useConnectedWeb3Context, useContracts } from '../../../../hooks'
+import {
```

Comment:
Got error when trying to sell max shares in ETH 
![image](https://user-images.githubusercontent.com/4677914/105993948-8de99f80-60d9-11eb-80e6-71dec6da6656.png)

Reply:
Steps to reproduce.
Go to market /#/0xf64d13a5aea7b1d20cbae747898f6099e77a7370 on Rinkeby.
Buy outcomes using ETH.
Try to sell max amount of outcomes.

Reply:
I have seen this error when developing the compound feature as well so might not be directly related to this particular PR but to one of the previous commits in this chain, so feel free to tackle this in a separate issue if you feel that this is not within the scope of this PR.

Reply:
hmmmm I couldn't reproduce this 🤔 def make a new issue so we explore though. I think this has happened before and it turned out that the subgraph was out of sync but unsure if this is the same or not.